RegExp.make.js

```diff
@@ -37,7 +37,7 @@ RegExp.make = (function () {
     if (0x20 <= n && n <= 0x7e) {
       return String.fromCharCode(n).replace(UNSAFE_CHARS_CHARSET, '\\$&');
     }
-    var hex = n.toString(16);
+    const hex = n.toString(16);
```

Comment:
Ah, I'd edited on the web interface. I didn't realize you're using closure compiler :-/ I'll try running check.sh locally before updating.